circle.yml

```diff
@@ -426,6 +460,7 @@ workflows:
       - build-gcc-32bit
       - build-cmake-min
       - build-macos
+      - build-windows
```

Comment:
The AppVeyor CI had 4 windows builds. We don't have to keep all of them, but having 32-bit build may be useful.

Reply:
Yeah, the 32-bit can be easily added back I suppose with replacing `-Arch amd64` with `-Arch x86`.

Not sure if circleci has images with the 3 older visual studio versions.